debug-panel/package.json

```diff
@@ -24,8 +24,8 @@
     "web-vitals": "^3.3.1"
   },
   "scripts": {
-    "start": "yarn --cwd ../networking build && react-scripts start",
-    "build": "yarn --cwd ../networking build && react-scripts build",
+    "start": "yarn --cwd ../networking build && mkdir -p ./node_modules/@righton/networking && rsync -r ../networking/lib ../networking/src ../networking/package.json ./node_modules/@righton/networking && react-scripts start",
+    "build": "yarn --cwd ../networking build && mkdir -p ./node_modules/@righton/networking && rsync -r ../networking/lib ../networking/src ../networking/package.json ./node_modules/@righton/networking && react-scripts build",
```

Comment:
@drewjhart This is what was missing, the changes in networking wasn't being updated under node_modules because we weren't copying it so it was using an older version. I've updated Play version to also matches here to both build and copy the latest networking.